packages/fluentui/docs/src/examples/components/Breadcrumb/Usage/BreadcrumbExample.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Breadcrumb } from '@fluentui/react-northstar';
+
+const BreadcrumbExample = props => (
+  <Breadcrumb>
+    <Breadcrumb.Item>Home</Breadcrumb.Item>
```

Comment:
```suggestion
    <Breadcrumb.Item><a href="">Home</a></Breadcrumb.Item>
```

Can we do this to the other items ( maybe except by the last one )

Reply:
Fixed by commit 
[feat(Breadcrumb): default example + icon divider example](https://github.com/microsoft/fluentui/pull/14926/commits/7fd8750359ab838b30235fc4ab2bc2b3eb35beb6)